spec/FeatureSpec.js

```diff
@@ -0,0 +1,93 @@
+describe('BowlingApp',function(){
+
+  beforeEach(function(){
+    jasmine.getFixtures().fixturesPath = '.';
+    loadFixtures('index.html');
+    $.holdReady(false);
+    scorecard = new ScoreCard();
+  });
+
+  it("displays score in table when a button is clicked", function() {
+    $("button[data-pins=2]").click();
```

Comment:
```$("button[data-pins=2]").click();``` <-- this is repeated a lot. Could you have a helper function to clean this up? The interface might look like: ```clickPin(3)```.